docs/advanced_topics/images/standardizing_renditions.rst

```diff
@@ -125,14 +124,14 @@ In your code
 As before, this assumes that you have your filter specs in the ``settings.py``.
 
  .. code-block:: python
- 
+
     from django.conf import settings
 
-    # Let's say you want to generate a tiny, low quality, version of an image.
+    # Lets say you want to generate a tiny, low quality, version of an image.
```

Comment:
In this case "Let's" was the correct version :)

Reply:
sometimes this english not the bestest one haha

thanks for finding that one. fixed with latest commit